### ISSUE TITLE
Window SHGC

### ISSUE THREAD

**Issue by khaddad** [canmet-energy/btap_tasks#276]
### Description
Write a decription of the issue. If it is a bug, please list all steps and attach a sample file, or sample ruby code that exhibits the bug. 

### Approach
After reviewing the issue, decribe your solution. If including git hyperlinks, ensure hyperlinks to existing code are to a version so it does not change with updates to git. 

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	

|CEO  |P-004001.001 |Cost-Effective Pathways for NZE, NZE Ready, and GHG Neutral Buildings |Phylroy Lopez

**Comment by mlubun** [comment]
SHGC Costing option
[SHGC thought.docx](https://github.com/canmet-energy/btap_tasks/files/4800791/SHGC.thought.docx)

**Comment by phylroy** [comment]
SHGC:
Obtaining costs for coatings to glass to reduce SHGC is going to be difficult.  Glass and IGU manufacturers (PPG, Viracon, Guardian, Cardinal, …) do not reveal their pricing to the general public.  They either negotiate directly with large buyers or have their dealer network negotiate with buyers (window and curtainwall companies).  
Therefore, the question is:  Where do we get the costs of different coatings that provide different levels of SHGC?
RSMeans may have a possible solution.
Below are the costs per ft2 for various films for glazing,  RSMeans does their costing related to VT .

![shgc](https://user-images.githubusercontent.com/5103270/88403395-7f24d780-cd9a-11ea-8653-4faff821a592.png)

RSMeans, also has a section on Insulating Glass with various properties.  I currently do not use this data, as I am costing complete window systems (glazing+framing+mullions+hardware) using the RSMeans Assembly method (complete window, glass door, skylight curtainwall).  The Assembly method does 2 types for everything (standard and insulating glass).  I was told that insulating glass is basically single or double glazing with a standard low e coating on surface 2  (for single glazing) or 3.   I have selected the insulating glass option for every window, curtainwall and skylight.  For glass doors, the insulating option is not available; as they are all tempered glass doors.
We could switch to building windows on a component basis to get more options related to thermal performance or interpolate based upon what RSMeans is publishing in the Assembly method.
![shgc2](https://user-images.githubusercontent.com/5103270/88403478-9cf23c80-cd9a-11ea-91ce-32d0450ab489.png)
Below are the curtainwall options.  I only use the insulating glass curtainwall option (B20202200042 – B20202201700.  However, looking at the difference in cost different coatings could give us an
![shgc3](https://user-images.githubusercontent.com/5103270/88403931-2d308180-cd9b-11ea-8961-7b1b5efd801c.png)
 approximation on coating coats.  
B202022001400 is single glazing clear ¼ in. float glass with a low e coating on surface 2   material = $18.80 ft2
B202022001500 is single glazing tinted ¼ in. float glass with a low e coating on surface 2   material = $26.50 ft2 or $7.70 ft2 for tinting
B202022001600 is single glazing  ¼ in. float glass with a special low e coating on surface 2   material = $31.50 ft2 or $12.70 ft2 for advanced low e coating
B202022001700 is single glazing  ¼ in. float glass with a special low e coating on surface 2 and tinting   material = $36.0 ft2 or $17.20 ft2 for advanced low e coating and tinting.
In the absence of coating costs, we could use these differences as the cost of the coatings.  The labour stayed the same.  When I worked in the curtainwall industry (33 years ago), we generally charged $10/ft2 for  low e coating, $10-15/ft2 for most tints, and $8/ft2 for argon fill.  I do not know what our markup was.  Those prices were when low e and argon were in their infancy, so I expect these prices to be reduced, in addition to the low e coatings being so much better than 33 years ago.

**Comment by phylroy** [comment]
Background on BTAP implementation. 

To simplify analysis and also to use the faster fenestration algorithms, BTAP has decided to use the simple glazing system. This also minimizes the inputs required for analysis. The windows are models as a single pane that exhibits the conductance values of mroe complex fenestration systems. The entry is defined like the example below. 

```
OS:WindowMaterial:SimpleGlazingSystem,
  {0cdf3593-c88a-4939-9a4a-9ccade8f882c}, !- Handle
  SimpleGlazing:U=0.295 SHGC=0.490,       !- Name
  2.95,                                   !- U-Factor {W/m2-K}
  0.49,                                   !- Solar Heat Gain Coefficient
  0.21;                                   !- Visible Transmittance
```

**Comment by phylroy** [comment]
The method to change the SHGC was added years ago to BTAP.. but was never invoked.. The argument was always set to 'nil' and accepted the defaults for the SHGC.  
b30092a22615d47c1f28bf039e5b112f4d511/lib/openstudio-standards/btap/envelope.rb#L922

**Comment by phylroy** [comment]
@ckirney @mlubun  I've noticed that @ckirney has a formula based on HDD that sets the SHGC/Tsol.  The current value for btap NECB vintages is 0.6.. Should we adopt the same SHGC as the 1980 vintages or keep the current values? https://github.com/NREL/openstudio-standards/blob/nrcan/lib/openstudio-standards/standards/necb/BTAPPRE1980/data/window_characteristics.json#L14

**Comment by phylroy** [comment]
SHGC has been implemented and pushed to standards and btap-costing as of today.  Waiting on costing data.
